main.js

```diff
@@ -60,6 +61,9 @@ var cli_opts = nopt(known_opts, shortHands);
 
 var cmd = cli_opts.argv.remain.shift();
 
+cli_opts.project = convertToRealPathSafe(cli_opts.project);
```

Comment:
Making them absolute here won't help when using cordova-lib via `cordova` or via `cordova-lib` directly. Could you look and see if you can resolve them outside of main.js?